.github/workflows/autofix.yml

```
@@ -24,4 +24,4 @@ jobs:
       - run: ncc build index.js
         env:
           NODE_OPTIONS: "--openssl-legacy-provider"
-      - uses: autofix-ci/action@551dded8c6cc8a1054039c8bc0b8b48c51dfc6ef
+      - uses: autofix-ci/action@v1.3.1
```

Sorry, I mean to keep the hash but for a release rather than main

```suggestion
      - uses: autofix-ci/action@2891949f3779a1cafafae1523058501de3d4e944  # v1.3.1
```